docs/install/macos.md

```diff
@@ -28,26 +30,30 @@ all users.
    It is recommended that you re-login, so that your environment knows about `elan`.
    (Alternatively, type `source $HOME/.elan/env` to update the current terminal.)
 
-### Installing mathlib supporting tools
 
-In order to use mathlib supporting tools, you need to [get python](https://www.python.org/downloads/) first.
-Then, at a terminal, run the command
+Installing mathlib supporting tools
```

Comment:
Why removing the line about getting python? Is it installed on all MacOS?

Reply:
No, but `remote-install-update-mathlib.sh` successfully prompts to install python if needed, so there's no need to install it beforehand.